setup.cfg

```diff
@@ -29,18 +29,21 @@ include_package_data = True
 packages = find:
 install_requires =
     django
+    djangorestframework
 tests_require =
     pytest
     pytest-django
     tox
     isort
-
+    black
 [options.extras_require]
 tests =
+    drf-spectacular
     pytest
     pytest-django
     tox
     isort
+    black
 pep8 = flake8
```

Comment:
I see that that flake8 is included here, but it isn't used in the github actions or in Tox. I don't know if it's on purpose, but I thought I'd mention it just in case :)

Reply:
flake8 is here because it's included in the maykin app template, so I left it as it is. Not a lot of our apps use flake8, so I didn't include it into github actions, if it's needed it could be added in the next PRs :)

Reply:
yeah, I'm usually a bit hesitant enabling flake8. We can enable it later, but it doesn't hurt to have it installed.